commands/src/main/java/org/wildfly/extras/creaper/commands/ra/RemoveResourceAdapter.java

```diff
@@ -0,0 +1,35 @@
+package org.wildfly.extras.creaper.commands.ra;
+
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+
+/**
+ * Remove existing resource adapter
+ */
+public final class RemoveResourceAdapter implements OnlineCommand {
+    private final String resourceAdapterId;
+
+    /**
+     * @param resourceAdapterId name of resource adapter
+     */
```

Comment:
It's not a problem, feel free to keep it. But frankly, this kind of javadoc helps noone -- it merely restates what is obvious from the code.